.github/workflows/sanity.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+
+on:
+    pull_request:
+        types: [ opened,edited,reopened,synchronize ]
```

Comment:
I think we can omit `edited`? It would be possible to re-run sanity tests by closing and opening PR, edited will trigger also when someone edits title right?

Reply:
yes I think so, I can delete it
For now, this PR is still in WIP, it's not ready for review yet.